package/MDAnalysis/core/universe.py

```diff
@@ -213,7 +214,19 @@ class Universe(object):
         master ConnectivityGroups for each connectivity type
 
     """
-
+    @due.dcite(BibTeX(("""
```

Comment:
The citations should go into the top level `MDAnalysis/__init__.py` – or what is your rationale for putting them here?

Reply:
Actually, ignore my comment. It makes sense to add the citations to `Universe`. 

However, is there a way to associate these citations globally with the library because someone might just use `MDAnalysis.lib.formats` but we would still want them to cite us.